### ISSUE TITLE
All new PRs are blocked on failing Chrome stability job

### ISSUE THREAD

**Issue by foolip** [web-platform-tests/wpt#13112]
Lots of recently opened PRs are blocked on failing Travis checks.

This is the earliest I can find, from https://travis-ci.org/web-platform-tests/wpt/jobs/430913594:
```
Traceback (most recent call last):
  File "./wpt", line 5, in <module>
    wpt.main()
  File "/home/travis/build/web-platform-tests/wpt/tools/wpt/wpt.py", line 129, in main
    rv = script(*args, **kwargs)
  File "/home/travis/build/web-platform-tests/wpt/tools/ci/check_stability.py", line 276, in run
    wpt_kwargs = setup_wptrunner(venv, **wpt_kwargs)
  File "/home/travis/build/web-platform-tests/wpt/tools/wpt/run.py", line 458, in setup_wptrunner
    check_environ(kwargs["product"])
  File "/home/travis/build/web-platform-tests/wpt/tools/wpt/run.py", line 140, in check_environ
    raise WptrunError(message)
tools.wpt.run.WptrunError: Missing hosts file configuration. Run
./wpt make-hosts-file | sudo tee -a /etc/hosts
```

Affected PRs:
https://github.com/web-platform-tests/wpt/pull/13092
https://github.com/web-platform-tests/wpt/pull/13093
https://github.com/web-platform-tests/wpt/pull/13094
https://github.com/web-platform-tests/wpt/pull/13095
https://github.com/web-platform-tests/wpt/pull/13103
https://github.com/web-platform-tests/wpt/pull/13104
https://github.com/web-platform-tests/wpt/pull/13105
https://github.com/web-platform-tests/wpt/pull/13106
https://github.com/web-platform-tests/wpt/pull/13109
https://github.com/web-platform-tests/wpt/pull/13110

Failing status checks for other reasons:
https://github.com/web-platform-tests/wpt/pull/13107
https://github.com/web-platform-tests/wpt/pull/13108

Not affected because "No tests changed" in log:
https://github.com/web-platform-tests/wpt/pull/13096
https://github.com/web-platform-tests/wpt/pull/13097
https://github.com/web-platform-tests/wpt/pull/13098
https://github.com/web-platform-tests/wpt/pull/13099
https://github.com/web-platform-tests/wpt/pull/13100
https://github.com/web-platform-tests/wpt/pull/13101
https://github.com/web-platform-tests/wpt/pull/13102

**Comment by foolip** [comment]
I sent https://github.com/web-platform-tests/wpt/pull/13111 because https://github.com/web-platform-tests/wpt/pull/13091 was the PR immediately before the problems started happening, and it says "[ci skip]". I don't know if it is really the cause yet.

**Comment by foolip** [comment]
https://github.com/web-platform-tests/wpt/pull/13113 by @jgraham will probably fix this.

If that's right, then https://github.com/web-platform-tests/wpt/pull/13091 was the cause. @jdm, did that PR bypass Travis, is that what "[ci skip]" means? If so, can you file an issue for Servo syncs to not bypass checks?

**Comment by jdm** [comment]
Yes, this is the perfect moment to pull the switch on our new system that will not bypass them.

**Comment by foolip** [comment]
@jdm, sweet, is that ready for action right now?

**Comment by jdm** [comment]
As far as I know, yes.

**Comment by foolip** [comment]
Reopening until it's clear that new PRs aren't affected, and the listed affected ones recovered.

**Comment by foolip** [comment]
OK, it's clear now the problem is fixed. @jdm, I'll close this, if you need another issue to track "ci skip" please file one :)
